### ISSUE TITLE
Decouple System from DBpedia

### ISSUE THREAD

**Issue by SiC42** [Berberer/SurniaQA#4]
After #3 is merged, the system won't really be productive ready, as there are still some systems that need to be replace/reworked to get SurniaQA to work correctly again.

One of the problems is the build in dependency of DBpedia. #2 is the first issue to this.

Another issue is that the systems queries DBpedia, mainly found in the [AnswerContainer](https://github.com/dice-group/SurniaQA/blob/refactor/decouple/src/main/java/de/upb/ds/surnia/qa/AnswerContainer.java)-class. In the future the system should provide queries solely based on the Queries.json and the queried endpoint should be as freely to choose as possible.
